### PR TITLE
Add students (Yong Chen)

### PR DESCRIPTION
**Pull request by EthanoicAcetate**

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://EthanoicAcetate.github.io/gci15.fossasia.org

File: _data/students.yml [hunk 1]
```diff
@@ -381,7 +381,7 @@
   github: Eclipse-Dominator
   image: LZQ_img.png
   twitter: Doomgrammar
-  bio: Believe in the impossible!
+  bio: Believe in the impossibLE!
```
Comment (roonyh) [inline _data/students.yml]: Lets not change this other students details :)

**Comment by roonyh** [comment]
Good job. Just remove the changes you are making in another students bio.

You can use the `amend` flag to update the previous commit instead of adding a new one. `git commit --amend`

And `git push -f`

**Comment by prwatch** [comment]
DEMO: http://EthanoicAcetate.github.io/gci15.fossasia.org

**Comment by prwatch** [comment]
DEMO: http://EthanoicAcetate.github.io/gci15.fossasia.org

**Comment by mananwason** [comment]
Nice @EthanoicAcetate !!

**Pull request merged**